### ISSUE TITLE
Solution split

### ISSUE THREAD

**Issue by xperiandri** [OrchardCMS/Orchard2#920]
VS 15.3 on my Surface Pro 3 is extremely unresponsive with so many projects in solution.
Even lightweight solution loading does not help too much.

I propose to split solution into manageable pieces.
What is the best approach in your opinion.

**Comment by sebastienros** [comment]
The end users will use the nuget packages. However for the ones who prefer to work with the source, or the core devs I agree. @Jetski5822 will probably have some idea.

**Comment by infofromca** [comment]
#964

**Comment by xiaoyc** [comment]
+1 ,loading orchard project consume to much memory.

**Comment by flq** [comment]
Indeed, opening orachrd core for the first time - 127 Projects. Fan howling for minutes.

Chances are this Project is violating the **"Projects organize deployment artefacts, not architectural boundaries"** rule. At first glance I saw a Project with one(!) file in it. Task and TaskAbstractions separated. These are probably namespaces. I think one could save man-years of developer time by analyzing what are Independent deployment artefacts and merging the Projects accordingly.

**Comment by Piedone** [comment]
The current approach to this is solution filters, what @hishamco attempted here: https://github.com/OrchardCMS/OrchardCore/pull/10436.

**Comment by hishamco** [comment]
@Piedone if we planned to do it, I will submit a PR again

**Comment by Piedone** [comment]
Duplicated by https://github.com/OrchardCMS/OrchardCore/issues/5660.
